include/common/msgType.h

```diff
@@ -259,6 +264,25 @@ struct s1apMsg_plus_raw_nas {
 	struct STMSI s_tmsi;
 }__attribute__ ((packed));
 
+struct erabSuResp_Q_msg {
+    int s1ap_enb_ue_id;
+    int s1ap_mme_ue_id;
+    erab_setup_list erab_su_list;
+    erab_failed_to_setup_list erab_fail_list;
+}__attribute__ ((packed));
+
+struct dedicatedBearerContextAccept_Q_msg {
```

Comment:
Should we move NAS messages to nas specific header file ? ...

Reply:
Yes, msg structs can be moved to protocol specific header files. Shall we take that as a separate activity?

Reply:
Sure. No worry. if not big time effort do it now. I will review your code quickly to take to final stage quickly. We had lot of trouble in separating this code.